providers/azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/features/NetworkInterfaceCardApiLiveTest.java

```diff
@@ -105,6 +105,12 @@ public void listNetworkInterfaceCards() {
       assertTrue(nicList.contains(api().get(nicName)));
    }
 
+   @Test(dependsOnMethods = "createNetworkInterfaceCard")
+   public void listAllNetworkInterfaceCards() {
```

Comment:
Make the delete test depend on this method too, to avoid deleting the networks first, causing this test to fail.

Reply:
Good catch, that test could definitely fail.